Project.toml

```diff
@@ -19,14 +19,14 @@ CUDA = "1, 2, 3"
 ChainRulesCore = "0.8, 0.9, 0.10, 1"
 Flux = "0.11, 0.12, 0.13"
 JOLI = "0.7, 0.8"
-NNlib = "0.7"
-Wavelets = "0.9"
+NNlib = "0.7, 0.8"
```

Comment:
I believe so. The main problem used to be with the convdims wrapper here but that got fixed a few PRs ago
https://github.com/slimgroup/InvertibleNetworks.jl/blob/master/src/InvertibleNetworks.jl#L28 

Im pretty sure that master already has 0.8. This is showing up as a change because of a messy rebase I did.

Reply:
Oh ok. Just didn't remember if that change was backward compatible with 0.7.